root/etc/e-smith/templates/etc/postfix/main.cf/40enableTLS

```diff
@@ -0,0 +1,27 @@
+
+#
+# TLS
+#
+
+# SMTPD
+smtpd_tls_security_level = may
+smtpd_tls_auth_only = no
+smtpd_tls_key_file = /etc/postfix/postfix.key
+smtpd_tls_cert_file = /etc/postfix/postfix.crt
```

Comment:
SSL certificates are currently set when mail-server is installed. Postfix relies on dovecot certs for SMTP/AUTH over TLS `smtpd_tls_auth_only = yes`.

See also http://www.postfix.org/TLS_README.html#server_tls_auth

As Postfix now requires a certificate also for sending messages (i.e. in a gateway-only use case),  we must decide how to split the new configuration, and we must do in a way that is compatible with both mail and mail2 implementations.